vignettes/geologic-map.Rmd

```diff
@@ -100,95 +100,74 @@ library(sf)
 
 First, we can use the package `rnaturalearth` to get the outlines of the USA, Canada and Mexico to use as a background. 
 
-```{r load countries, eval = TRUE}
-# Load background information
+```{r load-countries, eval = TRUE}
+# Load country outlines
 n_a <- ne_states(country = "united states of america", returnclass = "sf")
```

Comment:
I got a warning here telling me I also needed to install `rnaturalearthhires`. What's with this? If all users will need it then it should also be downloaded/added earlier.

Reply:
We unfortunately need the states to be able to show state lines, as otherwise the small map will just be a grey area. Any ideas what else we could do to fix this? Or should I just put rnaturalearthhires in the installs?